### PR TITLE
feat: add code style settings doc for IntelliJ Idea

### PR DESCRIPTION
**Pull request by hsiliev**
Adds code style settings import guide

### CODE REVIEW

**Review by edgarvonk — CHANGES_REQUESTED** [review]
Nice! We do also have IntelliJ docs already in our `INSTALL.md` (`### Run ZAC in IntelliJ`) and link from INSTALL.md to ths file. 

I think it makes sense to move those to this new file as well. I'm a little reluctant however with all this IntelliJ documentation because we are an open source project and I think it is a little strange to sort of 'push' a specific IDE but oh well. It does make our life easier so it's fine.

Also so far we have added a link to all development docs in our `README.md` because that is the index/starting page if you use GitHub. I think good to add a link there to this new file.

File: docs/development/intellij.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Code style settings
+
+## Java
+
+1. Open your Intellij Idea settings
+2. Navigate to Editor -> Code Style -> Java
+3. From Scheme options select Import Scheme -> Eclipse XML Profile
+![code_style_settings.png](./attachments/images/code_style_settings.png)
+4. Open `./config/Dimpact.xml` from the project root
```
Comment (edgarvonk) [inline docs/development/intellij.md:9]: Oh, yes I would not name this `Dimpact.xml`. This open source project should really stand on it's own and be independent of 'Dimpact' I would rename this just `zac.xml`.

**Review by bas-info-nl — APPROVED** [review]

**Comment by hsiliev** [comment]
> Nice! We do also have IntelliJ docs already in our `INSTALL.md` (`### Run ZAC in IntelliJ`). I think it makes sense to move those to this new file as well and link from INSTALL.md to this file.
> 
> I'm a little reluctant however with all this IntelliJ documentation because we are an open source project and I think it is a little strange to sort of 'push' a specific IDE but oh well. It does make our life easier so it's fine.
> 
> Also so far we have added a link to all development docs in our `README.md` because that is the index/starting page if you use GitHub. I think good to add a link there to this new file.

Merged/Extracted IDE specific stuff in a new file

**Review by edgarvonk — APPROVED** [review]
very nice :-)

**Pull request merged**